p8s.go

```diff
@@ -36,9 +38,16 @@ var (
 	MetricsURI string
 )
 
-func addRequest(labels map[string]string, bytes int) {
+func addRequest(labels map[string]string, logline map[string]interface{}) {
+	bytes := getBytes(logline)
+
 	requestsTotal.With(labels).Inc()
 	bytesTotal.With(labels).Add(float64(bytes))
+
+	// Count text/html 2XX requests as page-views
+	if strings.HasPrefix(fmt.Sprintf("%v", logline["status"]), "2") && logline["content_type"] == "text/html" {
```

Comment:
Seems like a good candidate for a function `has2XXStatus(...)`, again for content_type as well.